internal/tools/go.sum

```diff
@@ -16,6 +16,7 @@ github.com/stretchr/testify v1.10.0 h1:Xv5erBjTwe/5IxqUQTdXv5kgmIvbHo3QQyRwhJsOf
 github.com/stretchr/testify v1.10.0/go.mod h1:r2ic/lqez/lEtzL7wO/rwa5dbSLXVDPFyf8C91i36aY=
 go.opentelemetry.io/build-tools/chloggen v0.22.0 h1:QLy1ox4iyF/lVTlRnN7B+1AFJzczvlwPNVKjYvwmc2c=
 go.opentelemetry.io/build-tools/chloggen v0.22.0/go.mod h1:wJ8tJY5bURruxByT9gaaaGxjr1FoK8bUl/FiE+yQTfE=
+go.opentelemetry.io/build-tools/chloggen v0.23.1 h1:vc7BCQ43OtEr9onDU/0igWwM08ocn8AzxX+0BXG9OPo=
```

Comment:
renovate bot didn't add this line in the https://github.com/open-telemetry/semantic-conventions/pull/2227